dj_cqrs/utils.py

```diff
@@ -33,3 +33,37 @@ def get_expires_datetime():
         message_ttl = DEFAULT_CQRS_MESSAGE_TTL
 
     return timezone.now() + timedelta(seconds=message_ttl)
+
+
+def get_delay_queue_max_size():
+    """Returns delay queue "waiting" messages number.
+
+    :return: integer instance, None if infinite
+    :rtype: int
+    """
+    replica_settings = settings.CQRS.get('replica', {})
+    max_size = DEFAULT_DELAY_QUEUE_MAX_SIZE
+    if 'delay_queue_max_size' in replica_settings:
+        max_size = replica_settings['delay_queue_max_size']
+
+    if max_size is not None and max_size <= 0:
+        logger.warning(
+            "Settings delay_queue_max_size=%s is invalid, using default %s.",
+            max_size, DEFAULT_DELAY_QUEUE_MAX_SIZE,
+        )
+        max_size = DEFAULT_DELAY_QUEUE_MAX_SIZE
+    return max_size
+
+
+def get_rabbitmq_prefetch_count():
```

Comment:
Hm, I don't think that it makes sense to put something transport specific (rabbit_mq prefetch count) into common utils.
I would suggest putting it somewhere closer to transport implementation.

Reply:
In general all utils now are rabbitmq specific.
I don't like the idea of keeping every function as a static/class method.
My suggestion - rename it to get_prefetch_count.

Reply:
Ok